packages/foam-vscode/src/features/tags-tree-view/index.ts

```diff
@@ -62,23 +63,51 @@ export class TagsProvider implements vscode.TreeDataProvider<TagTreeItem> {
   }
 
   getTreeItem(element: TagTreeItem): vscode.TreeItem {
+    if (element.title.indexOf(TAG_SEPARATOR) > -1) {
+      return;
+    }
     return element;
   }
 
   getChildren(element?: Tag): Thenable<TagTreeItem[]> {
     if (element) {
-      const references: TagReference[] = element.notes
+      const tagChilds: TagTreeItem[] = this.tags
```

Comment:
minor: the plural of child is children 😅 
I would suggest naming this something like `nestedTagItems`

Reply:
Not sleeping enough with kids is not helping my English ;-)